docs/UserGuide.md

```diff
@@ -118,31 +118,54 @@ Upon completing this section, you should be able to explore SoConnect on your ow
 
 ### 6.2. Key Definitions
 
+### 6.2.1 Command
+
+Command is the action you want SoConnect to perform. Most commands require [parameters](#622-parameter) after the command so that SoConnect has the required information to perform the action. 
+
+e.g. `help`
+
+### 6.2.2 Parameter
+
+A parameter has a prefix and information which are written together without a space in between. The information are words in `UPPER_CASE` and are to be provided by you. The prefix, which ends with a `/`, is specified before the information and is what specifies what type of information you are providing SoConnect with. Do note that some commands only requires the prefix as a parameter.  <br>
+
+e.g. `n/NAME`
+
 ### 6.3. Command Format
 
-**:information_source: Notes about the command format:**<br>
+To use SoConnect features, you would need to type the [command](#621-command) that you would like to use along with the [parameters](#622-parameter) it requires. If a parameter is optional, it will be wrapped with square brackets. e.g. `[e/EMAIL]` <br>
```

Comment:
```suggestion
In the command box, you need to type the [command word](#621-command) along with the required [parameters](#622-parameter). You can leave the parameter field empty if it is optional, which is indicated by wrapped square brackets in the command format, e.g. `search [e/EMAIL]` <br>
```

Reply:
The commands, parameters, formats, etc. are consistently linked, which might not turn out good. I think general rule of thumb is one per section is enough (or at least if the user can still see the same word in the same eyespan then should be no need to link)

Reply:
Ok I shall remove the extra ones. Another point is that I think that it should still be `command` instead of `command word` as a lot of commands have subcommands/more than one word and it would be simpler to refer to all of them as `commands`.